.buildkite/pipeline.yml

```
@@ -132,8 +133,9 @@ steps:
       buildkite-agent artifact download "*.nupkg" .
       buildkite-agent artifact download "*\\RELEASES" .
 
-      echo "--- :node: Generating Release Manifest"
       .buildkite/commands/install-node-dependencies.sh
```

Same as a bove, I wonder if we could include this command in its own section:
```suggestion
      echo "--- :node: Installing Node dependencies"
      .buildkite/commands/install-node-dependencies.sh
```